components/RecipeForm/RecipeForm.tsx

```diff
@@ -269,15 +380,9 @@ export default function RecipeForm({ onAddRecipe }: OnAddRecipeType) {
             })}
           </StyledCategoryContainer>
         </StyledInputElement>
-        <StyledButtonWrapper>
-          <Button
-            variant='$submit'
-            type='submit'
-            aria-label='submit new recipe'
-          >
-            Submit Recipe
-          </Button>
-        </StyledButtonWrapper>
+
+        {/* Action Buttons */}
+        <StyledButtonWrapper>{renderButtons()}</StyledButtonWrapper>
```

Comment:
You could use the Button component to render the button. Why do you need an extra function here? (:

Reply:
I'll create a `FormButtons` component and extract the logic to that newly created component, you're right. Thanks!